src/lib/detection_algorithms/cntkfastrcnn/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "main" : "main.js",
+    "dependencies": {
+        "cntk-fastrcnn": "^0.2.2"
```

Comment:
@aribornstein  - While it make since to have this dependency here for the module, it's worth mentioning that since these modules aren't part of the node_modules directory, we won't have [npm install] running here recursively. We might want to use something like: https://github.com/emgeee/recursive-install eventually.
For now we'll just need to add an explanation at the README that you should add the modules with your extension. (which is OK)